dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -72,6 +73,25 @@ public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true)
         return lambdaEventDAO.findByOrganization(organization, offset, limit, authorizedRepos);
     }
 
+    @GET
+    @Timed
+    @UnitOfWork(readOnly = true)
+    @RolesAllowed({ "admin", "curator"})
+    @Path("/user/{userid}")
+    @Operation(operationId = "getUserLambdaEvents", description = "Get all of the Lambda Events for the given user.",
+            security = @SecurityRequirement(name = ResourceConstants.JWT_SECURITY_DEFINITION_NAME))
+    public List<LambdaEvent> getUserLambdaEvents(@Parameter(hidden = true, name = "user")@Auth User authUser,
+           @PathParam("userid") long userid,
+           @QueryParam("offset") @DefaultValue("0") Integer offset,
+           @DefaultValue("1000") @QueryParam("limit") Integer limit) {
+        final User user = userDAO.findById(userid);
+        final List<Token> githubTokens = tokenDAO.findGithubByUserId(userid);
+        if (githubTokens.isEmpty()) {
```

Comment:
Do we need this check? The events are in the table and retrievable regardless of whether they currently have their GitHub account linked or not, and it doesn't look like we need the GitHub token to do anything for this operation.